frontend/public/components/deployment.jsx

```diff
@@ -8,6 +9,8 @@ import { registerTemplate } from '../yaml-templates';
 import { Conditions } from './conditions';
 import { EnvironmentPage } from './environment';
 import { ResourceEventStream } from './events';
+import { ContainerTable } from './deployment-config';
```

Comment:
This is a nit, but it would be better to have ContainerTable defined here and imported by deployment-config.tsx. (Someday deployment configs might be replaced entirely by upstream deployments.)